fluss-flink/fluss-flink-common/src/main/java/com/alibaba/fluss/flink/source/FlinkSource.java

```diff
@@ -62,6 +65,8 @@ public class FlinkSource<OUT>
     private final boolean streaming;
     private final FlussDeserializationSchema<OUT> deserializationSchema;
 
+    @Nullable private final List<PushdownUtils.FieldEqual> partitionFilters;
```

Comment:
There is no need to declare the list array as @Nullable, as this would simplify the conditional checks in `applyPartitionFilter()` for `partitionFilters != null && !partitionFilters.isEmpty()`.